pyphare/pyphare/pharein/load_balancer.py

```diff
@@ -0,0 +1,45 @@
+#
+#
+
+from dataclasses import dataclass, field
+from . import global_vars as gv
+
+
+@dataclass
+class LoadBalancer:
+    # whether or not load balancing is performed
+    active: bool = field(default_factory=lambda: False)
+
+    # which way load is assessed
+    mode: str = field(default_factory=lambda: "nppc")
+
+    # acceptable imbalance essentially
+    tol: float = field(default_factory=lambda: 0.05)
+
+    # if auto, other values are not used if active
+    auto: bool = field(default_factory=lambda: True)
+
+    # if !auto these values are used if active
+    on_init: bool = field(default_factory=lambda: True)
+    every: int = field(default_factory=lambda: 1)
```

Comment:
> can we add a comment for

sure

`every` x timesteps is a cadence, so it's not in units of `timestep` but just int from 0 

> I guess these values concern the L0 level only?

yes, it's automatic otherwise